app/src/main/res/drawable/ic_key.xml

```diff
@@ -0,0 +1,6 @@
+<vector android:height="24dp" android:viewportHeight="280.062"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_key` appears to be unused
 Unused resources make applications larger and slow down builds.